core/src/test/java/org/apache/iceberg/hadoop/TestHadoopCommits.java

```diff
@@ -451,4 +455,39 @@ public void testConcurrentFastAppends(@TempDir File dir) throws Exception {
     Assertions.assertThat(Lists.newArrayList(tableWithHighRetries.snapshots()))
         .hasSize(threadsCount * numberOfCommitedFilesPerThread);
   }
-}
+
+  @Test
+  public void testCommitFailedToAcquire() {
+    table.newFastAppend().appendFile(FILE_A).commit();
+    Configuration conf = new Configuration();
+    LockManager lockManager = new NoLockManager();
+    HadoopTableOperations tops = 
```

Comment:
Naming nit: I think it's better to be a bit more verbose rather than truncate, since it's more readable imo. Instead of `tops` could we call it `tableOperations` like we do in other places? Also maybe `testCommitFailedToAcquireLock` for the test name?